apps/site/src/components/Sticker/BaseSticker.tsx

```diff
@@ -53,6 +58,7 @@ export default function Sticker({
 					filter: `drop-shadow(10px 14px 10px rgba(0, 0, 0, 0.2))`,
 				},
 				drag: true,
+				initial: { x: -width / 2, y: -height / 2 },
```

Comment:
Is this to center the sticker? Could it not be achieved in scss module?

Reply:
It should be possible to do by wrapping the sticker in another parent and transforming that. Not sure if that's a cleaner solution though.

Reply:
Yea I agree. Let's just stick with this.